go/cmd/vtbackup/vtbackup.go

```diff
@@ -431,6 +439,13 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 		return fmt.Errorf("not taking backup: replication did not make any progress from restore point: %v", restorePos)
 	}
 
+	// Re-enable redo logging.
+	if disabledRedoLog {
+		if err := mysqld.EnableRedoLog(ctx); err != nil {
+			return fmt.Errorf("failed to enable redo log: %v", err)
```

Comment:
What happens now? I think vtbackup will throw things away and retry, right? This could be problematic with the new flag to keep the temporary data though, right? I would think we might run out of disk space.

Reply:
By default `vtbackup` will delete the temporary files, but you're right that if we `--keep-temporary-files` then we could run out of disk space.